Google/Authorization.php

```diff
@@ -136,8 +137,9 @@ protected function getUserInfoByAccessToken(\Matomo\Dependencies\GoogleAnalytics
     private function getValidUri($uris)
     {
         $validUri = Url::getCurrentUrlWithoutQueryString() . '?module=GoogleAnalyticsImporter&action=processAuthCode';
+        $validUriFallback = SettingsPiwik::getPiwikUrl() . '?module=GoogleAnalyticsImporter&action=processAuthCode'; // Some MWP installs was not working as expected when using Url::getCurrentUrlWithoutQueryString()
```

Comment:
It's doesn't look like `SettingsPiwik::getPiwikUrl()` adds the PHP file, so the fallback URL won't match. I don't have a WP environment to confirm whether this is necessary, but it was when I disabled `$validUri` in my local non-WP instance.
```suggestion
        $validUriFallback = SettingsPiwik::getPiwikUrl() . 'index.php?module=GoogleAnalyticsImporter&action=processAuthCode'; // Some MWP installs was not working as expected when using Url::getCurrentUrlWithoutQueryString()
```

Reply:
@snake14 Dizzy suggested to append `index.php` conditionally if not present to cover all the cases